src/clr-angular/data/datagrid/datagrid-column-toggle.ts

```diff
@@ -131,6 +152,9 @@ export class ClrDatagridColumnToggle implements OnInit, OnDestroy {
     this.hideableColumnService.showHiddenColumns();
     this.allColumnsVisible = this.hideableColumnService.checkForAllColumnsVisible;
     this.columnToggleButtons.selectAllDisabled = this.allColumnsVisible;
+    if (this.allSelectedElement) {
```

Comment:
Honestly, I don't like this, as it's here only to make some tests pass, that are directly creating `new ClrDatagridColumnToggle(....... )` and not initializing the component as such.
But I also don't thinks it's worth prioritizing the update of these tests at the moment, just for the backported issue.